<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="marwa-sw" failures="2" tests="22" name="Regression test" time="6.606" errors="0" timestamp="2023-05-21T09:09:25 EET">
  <testcase classname="test.Class2" name="Class_02_test_01" time="0.002"/>
  <testcase classname="test.Class2" name="Class_02_test_02" time="0.002"/>
  <testcase classname="test.Class2" name="Class_02_test_03" time="0.002"/>
  <testcase classname="test.Class2" name="Class_02_test_04" time="0.003"/>
  <testcase classname="test.class1" name="Class_01_test_01" time="0.001"/>
  <testcase classname="test.class1" name="Class_01_test_01" time="0.001"/>
  <testcase classname="test.class1" name="Class_01_test_02" time="0.001"/>
  <testcase classname="test.class1" name="Class_01_test_02" time="0.0"/>
  <testcase classname="test.class1" name="Class_01_test_03" time="0.002"/>
  <testcase classname="test.class1" name="Class_01_test_03" time="0.003"/>
  <testcase classname="test.class1" name="Class_01_test_04" time="0.002"/>
  <testcase classname="test.class1" name="Class_01_test_04" time="0.0"/>
  <testcase classname="test.Class3" name="Class_03_test_01" time="0.002"/>
  <testcase classname="test.Class3" name="Class_03_test_02" time="0.001"/>
  <testcase classname="test.Class3" name="Class_03_test_03" time="0.001"/>
  <testcase classname="test.Class3" name="Class_03_test_04" time="0.001"/>
  <testcase classname="test.Class4" name="Class_04_test_01" time="0.001"/>
  <testcase classname="test.Class4" name="Class_04_test_02" time="0.001"/>
  <testcase classname="test.Class4" name="Class_04_test_03" time="0.002"/>
  <testcase classname="test.Class4" name="Class_04_test_04" time="0.014">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Class4.Class_04_test_04(Class4.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Class_04_test_04 -->
  <testcase classname="test.Login" name="user_logged" time="0.0"/>
  <testcase classname="test.Main3" name="Class_1_test_1" time="6.347">
    <failure type="org.openqa.selenium.NoSuchWindowException" message="no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=113.0.5672.126)
Build info: version: &amp;apos;4.8.1&amp;apos;, revision: &amp;apos;8ebccac989&amp;apos;
System info: os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [3f9bb1f81a0e8d79922a29b824fc9a17, get {url=http://www.google.com}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 113.0.5672.126, chrome: {chromedriverVersion: 113.0.5672.63 (0e1a4471d5ae..., userDataDir: C:\Users\AHMED~1.HEL\AppDat...}, goog:chromeOptions: {debuggerAddress: localhost:62729}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 3f9bb1f81a0e8d79922a29b824fc9a17">
      <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=113.0.5672.126)
Build info: version: '4.8.1', revision: '8ebccac989'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [3f9bb1f81a0e8d79922a29b824fc9a17, get {url=http://www.google.com}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 113.0.5672.126, chrome: {chromedriverVersion: 113.0.5672.63 (0e1a4471d5ae..., userDataDir: C:\Users\AHMED~1.HEL\AppDat...}, goog:chromeOptions: {debuggerAddress: localhost:62729}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 3f9bb1f81a0e8d79922a29b824fc9a17
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:297)
at test.Main3.Class_1_test_1(Main3.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- Class_1_test_1 -->
</testsuite> <!-- Regression test -->
